app/i18n/it-IT.po

```diff
@@ -0,0 +1,12 @@
+
+msgid "unavailable"
+msgstr "N/D"
+
+msgid "on"
+msgstr "ON"
+
+msgid "off"
+msgstr "OFF"
+
+msgid "exe"
+msgstr "ESEGUI"
```

Comment:
@maxcanna Does that text fit into the UI element? Did you test that (e.g. with Versa 2)?

Reply:
> @maxcanna Does that text fit into the UI element? Did you test that (e.g. with Versa 2)?

Yes, you're right.

What about using ▶️ ?
Take a look:

![image](https://user-images.githubusercontent.com/1881831/153722243-c68383e5-9a79-46ed-ba2e-ec39bd97393b.png)

Reply:
Is there no understandable way to express run or execute in Italian with just 3 letters?
If not, than the play icon is a good alternative solution.

Reply:
Yes I can truncate it to _ESE_ if you want to stay coherent

Reply:
Yes, truncation would be perfect

Reply:
ok, done